apps/admin-portal/src/components/applications/components/url-input-component.tsx

```diff
@@ -158,29 +185,33 @@ export const URLInputComponent: FunctionComponent<URLInputComponentInterface> =
             </Grid.Row>
             <Grid.Row className={ "urlComponentInputRow" }>
                 <Grid.Column mobile={ 14 } tablet={ 14 } computer={ 8 }>
-                    {
-                        validURL && !duplicateURL ?
-                            (
-                                <Input
-                                    fluid
-                                    value={ changeUrl }
-                                    onKeyDown={ keyPressed }
-                                    onChange={ handleChange }
-                                    placeholder={ placeholder }
-                                    icon={ <Icon name="add" onClick={ () => addUrl() } link/> }
-                                />
-                            ) : (
-                                <Input
-                                    fluid
-                                    error
-                                    value={ changeUrl }
-                                    onKeyDown={ keyPressed }
-                                    onChange={ handleChange }
-                                    placeholder={ placeholder }
-                                    icon={ <Icon name="add" onClick={ () => addUrl() } link/> }
-                                />
-                            )
-                    }
+                    <Input
+                        fluid
+                        error={ validURL && !duplicateURL ? false : true }
+                        focus={ keepFocus }
+                        value={ changeUrl }
+                        onKeyDown={ keyPressed }
+                        onChange={ handleChange }
+                        onBlur={ handleOnBlur }
+                        placeholder={ placeholder }
+                        action
+                    >
+                        <input/>
```

Comment:
Why do we need this empty input?

Reply:
When we write multiple actions then in the children we need to define this to get the input box. This is how defined in semantic https://react.semantic-ui.com/elements/input/#variations-actions

Reply:
ok